erizo_controller/erizoClient/src/utils/L.Resizer.js

```diff
@@ -1,460 +1,462 @@
-                            extractQuery(selector);
-                        }
-                    } else if (4 === rules[i].type) {
-                        readRules(rules[i].cssRules || rules[i].rules);
-                    }
-                }
-            }
+    const elements = query(selector);
+    for (let i = 0, j = elements.length; i < j; i += 1) {
+      elements[i] = setupElement(elements[i], {
+        mode,
+        property,
+        value,
+      });
+    }
+  }
+
+  const regex = /,?([^,\n]*)\[[\s\t]*(min|max)-(width|height)[\s\t]*[~$^]?=[\s\t]*"([^"]*)"[\s\t]*]([^\n\s{]*)/mgi;  // jshint ignore:line
+
+      /**
+       * @param {String} css
+       */
+  function extractQuery(originalCss) {
+    let match;
+    const css = originalCss.replace(/'/g, '"');
+    while ((match = regex.exec(css)) !== null) {
+      if (match.length > 5) {
+        queueQuery(match[1] || match[5], match[2], match[3], match[4]);
+      }
+    }
+  }
+
+      /**
+       * @param {CssRule[]|String} rules
+       */
+  function readRules(originalRules) {
+    let selector = '';
+    let rules;
+    if (!originalRules) {
+      return;
+    }
+    if (typeof originalRules === 'string') {
+      rules = originalRules.toLowerCase();
+      if (rules.indexOf('min-width') !== -1 || rules.indexOf('max-width') !== -1) {
+        extractQuery(rules);
+      }
+    } else {
+      for (let i = 0, j = rules.length; i < j; i += 1) {
+        if (rules[i].type === 1) {
+          selector = rules[i].selectorText || rules[i].cssText;
+          if (selector.indexOf('min-height') !== -1 ||
+                          selector.indexOf('max-height') !== -1) {
+            extractQuery(selector);
+          } else if (selector.indexOf('min-width') !== -1 ||
+                                 selector.indexOf('max-width') !== -1) {
+            extractQuery(selector);
+          }
+        } else if (rules[i].type === 4) {
+          readRules(rules[i].cssRules || rules[i].rules);
         }
+      }
+    }
+  }
+
+      /**
+       * Searches all css rules and setups the event listener
+       * to all elements with element query rules..
+       */
+  this.init = () => {
+    for (let i = 0, j = document.styleSheets.length; i < j; i += 1) {
+      readRules(document.styleSheets[i].cssText ||
+                        document.styleSheets[i].cssRules ||
+                        document.styleSheets[i].rules);
+    }
+  };
+};
+
+function init() {
+  (new L.ElementQueries()).init();
+}
+
+if (window.addEventListener) {
+  window.addEventListener('load', init, false);
+} else {
+  window.attachEvent('onload', init);
+}
+
+  // Only used for the dirty checking, so the event callback count is limted
+  //  to max 1 call per fps per sensor.
+  // In combination with the event based resize sensor this saves cpu time,
+  // because the sensor is too fast and
+  // would generate too many unnecessary events.
+const requestAnimationFrame = window.requestAnimationFrame ||
```

Comment:
The google-closure-compiler is complaining here of a dublicate const declaration, not sure why.